consensus/enclave/impl/src/lib.rs

```diff
@@ -937,13 +937,14 @@ impl ConsensusEnclave for SgxConsensusEnclave {
             validated_mint_config_txs,
             mint_txs,
         };
-        //
+
         // Form the block.
         let block = Block::new_with_parent(
             config.block_version,
             parent_block,
             root_element,
             &block_contents,
+            0,
```

Comment:
Because we don't have a timestamp value here yet we can't assert that it's 0 or supported and not zero, moving that change into #3733